doc/md/fundamentals/2-types/4-tuples.md

```diff
@@ -44,17 +44,17 @@ let second = ghost.1; // 25
 
 Tuples are useful for returning multiple values from a function without requiring a separate data structure.
 
-```motoko no-repl
+```motoko
 func getUserInfo() : (Text, Nat) {
     ("Ghost", 30);
-}
+};
 
 getUserInfo();
 ```
 
 ## Nesting tuples
```

Comment:
I would move the section `## Nesting tuples` to the end, since it relies on pattern matching of tuples.